mosquitto/DOCS.md

```diff
@@ -0,0 +1,155 @@
+# Home Assistant Add-on: Mosquitto broker
+
+## Installation
+
+Follow these steps to get the add-on installed on your system:
+
+1. Navigate in your Home Assistant frontend to **Supervisor** -> **Add-on Store**.
+2. Find the "Mosquitto broker" add-on and click it.
+3. Click on the "INSTALL" button.
+
+## How to use
+
+The add-on has a couple of options available. To get the add-on running:
+
+1. Start the add-on.
+2. Have some patience and wait a couple of minutes.
+3. Check the add-on log output to see the result.
+
+Create a new user for MQTT via the **Configuration** -> **Users (manage users)**.
+Note: This name cannot be `homeassistant` or `addon`, those are reserved usernames.
+
+To use the Mosquitto as a broker, go to the integration page and install the configuration with one click:
+
+1. Navigate in your Home Assistant frontend to **Configuration** -> **Integrations**.
+2. MQTT should appear as a dicovered integration at the top of the page
+3. Select it and check the box to enable MQTT discovery if desired, and hit submit.
+
+If you have old MQTT settings available, remove this old integration and restart Home Assistant to see the new one.
```

Comment:
It is too late doing this here.
The embedded broker must be disabled first (remove "mqtt:...." from configuration.yaml and then the mosquito installed and then integrated into home assistant.

Reply:
Home Assistant doesn't have an embedded broker anymore.

If you want to report an issue, please, report an issue instead of leaving comments on handled PRs. Thanks 👍